### DISCUSSION TITLE
Detach from original repo

### DISCUSSION THREAD

**Discussion by electron271** [logisim-evolution/logisim-evolution#1513] [Dev talk]
I personally feel that this repo should be detached from the original repo, so it doesn't view as a fork

**Comment by maehne** [comment]
@electron271: We agree! That the repo was displayed as a fork from another repo was an artefact from the necessary recovery of this repo after it got accidentally deleted (see issue #1512). Everything should be now back to normal. If you should encounter any other inconsistencies, please let us know!
